github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -45,13 +45,23 @@ presubmits:
           value: "kind-1.19-sriov"
         - name: "KUBEVIRT_NUM_NODES"
           value: "3"
+        - name: "RUN_KUBEVIRT_CONFORMANCE"
+          value: "true"
+        - name: "SONOBUOY_EXTRA_ARGS"
+          value: "--plugin-env kubevirt-conformance.E2E_FOCUS=SRIOV"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
         - "-c"
         - >
```

Comment:
`|` is better here, and then no need `&&` all over
see please https://yaml-multiline.info

Reply:
Done